frontend/src/components/Notification.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div class="notifications-container">
+        <transition 
+            enter-active-class="transition ease-out duration-200" 
+            enter-from-class="translate-x-full"
+            enter-to-class="translate-x-0" 
+            leave-active-class="transition ease-in duration-150"
+            leave-from-class="translate-x-0" 
+            leave-to-class="translate-x-full"
+        >
+            <div v-if="alert" class="notification" :class="`notification-${alert.type}`">
```

Comment:
Shouldn't there be a loop here somewhere to show multiple notifications? Also, I assume that this 'v-if' is here to allow for animations, right?

Reply:
At the moment, we don't have multiple notifications in the `alertStore`. I think we could rework it after merge this PR and yes the `v-if` starts the animation if something in the `alertStore`